includes/class-sensei-question.php

```diff
@@ -411,8 +411,13 @@ public function get_question_grade( $question_id ) {
      * @param $question_type
      */
     public static function load_question_template( $question_type ){
+		$template_loaded = Sensei_Templates::get_template( 'single-quiz/question_type-' . $question_type . '.php' );
 
-        Sensei_Templates::get_template  ( 'single-quiz/question_type-' . $question_type . '.php' );
+		if ( ! $template_loaded ) {
```

Comment:
Ah, there it is! I stared and started at the code trying to find the difference and just couldn't see it. Thanks for helping me see. :)